### PR TITLE
Tabular cli output

### LINKED ISSUE #39: Add tabular output to CLI

### PR DESCRIPTION
**Pull request by avishayt**
```
$ ./bin/flightctl get fleets
NAME
default
$ ./bin/flightctl get devices
NAME
75e85764af61c54a7df04a1740241ee2e764ee7ef975ae6316830d697426a908
89016cb25f18d3b4da9255bc2fbf9f934d44a709f9deaa1f148901c062aa3cd0
$ ./bin/flightctl get enrollmentrequests
NAME                                                                 APPROVED     REGION
75e85764af61c54a7df04a1740241ee2e764ee7ef975ae6316830d697426a908     true         default
89016cb25f18d3b4da9255bc2fbf9f934d44a709f9deaa1f148901c062aa3cd0     true         default
b0c9b755704cb13a57e62db96d3ca3899a5b17e233dba49b1024c61542991b93                  
```

Resolves #39

### CODE REVIEW

File: cmd/flightctl/main.go [hunk 1]
```diff
@@ -217,11 +226,21 @@ func RunGet(kind, name string, labelSelector string) error {
 				return fmt.Errorf("listing devices: %s", response.HTTPResponse.Status)
 			}
 
-			marshalled, err := yaml.Marshal(response.JSON200)
-			if err != nil {
-				return fmt.Errorf("marshalling device list: %v", err)
+			if output == "yaml" {
```
Comment (filanov) [inline cmd/flightctl/main.go:229]: maybe it should const?

File: cmd/flightctl/main.go [hunk 2]
```diff
@@ -217,11 +226,21 @@ func RunGet(kind, name string, labelSelector string) error {
...
+				marshalled, err := yaml.Marshal(response.JSON200)
+				if err != nil {
+					return fmt.Errorf("marshalling device list: %v", err)
+				}
+				fmt.Printf("%s\n", string(marshalled))
```
Comment (filanov) [inline cmd/flightctl/main.go:234]:
```suggestion
				fmt.Println(string(marshalled))
```

File: cmd/flightctl/main.go [hunk 3]
```diff
@@ -251,11 +270,27 @@ func RunGet(kind, name string, labelSelector string) error {
...
 
-			marshalled, err := yaml.Marshal(response.JSON200)
-			if err != nil {
-				return fmt.Errorf("marshalling enrollmentrequest list: %v", err)
+			if output == "yaml" {
+				marshalled, err := yaml.Marshal(response.JSON200)
+				if err != nil {
+					return fmt.Errorf("marshalling enrollmentrequest list: %v", err)
+				}
+				fmt.Printf("%s\n", string(marshalled))
```
Comment (filanov) [inline cmd/flightctl/main.go:278]:
```suggestion
				fmt.Println(string(marshalled))
```

Comment (avishayt) [reply cmd/flightctl/main.go]: Done

File: cmd/flightctl/main.go (hunk 2 above)
Comment (avishayt) [reply cmd/flightctl/main.go]: Done

File: cmd/flightctl/main.go [hunk 4]
```diff
@@ -13,16 +13,23 @@ import (
...
 	"github.com/flightctl/flightctl/internal/crypto"
 	"github.com/spf13/cobra"
+	"github.com/thoas/go-funk"
+
 	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	YAML = "yaml"
```
Comment (filanov) [inline cmd/flightctl/main.go:30]:
```suggestion
	yamlFormat = "yaml"
```
or something like that

File: cmd/flightctl/main.go [hunk 5]
```diff
@@ -251,11 +275,27 @@ func RunGet(kind, name string, labelSelector string) error {
 				return fmt.Errorf("listing enrollmentrequests: %s", response.HTTPResponse.Status)
 			}
 
-			marshalled, err := yaml.Marshal(response.JSON200)
-			if err != nil {
-				return fmt.Errorf("marshalling enrollmentrequest list: %v", err)
+			if output == "yaml" {
```
Comment (filanov) [inline cmd/flightctl/main.go:278]: should use the const

**Review by filanov — COMMENTED** [review]
Can you please add an example of the output?

File: cmd/flightctl/main.go (hunk 4 above)
Comment (avishayt) [reply cmd/flightctl/main.go]: Done

File: cmd/flightctl/main.go (hunk 5 above)
Comment (avishayt) [reply cmd/flightctl/main.go]: Done

**Review by filanov — APPROVED** [review]

**Pull request merged**